docs/collect/enable-configure.md

```diff
@@ -0,0 +1,60 @@
+<!--
+title: "Enable or configure a collector"
+description: "Every collector is highly configurable, allowing them to collect metrics from any node and any infrastructure."
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/collect/enable-configure.md
+-->
+
+# Enable or configure a collector
+
+While collectors start collecting right away if they find exposed metrics on their default endpoint, not all
```

Comment:
This sentence doesn't really make sense to me 🤔

Reply:
I tried to explain this better! It's a hard one.